src/track/trackmetadatataglib.cpp

```diff
@@ -785,6 +862,22 @@ void readTrackMetadataFromXiphComment(TrackMetadata* pTrackMetadata,
                 toQStringFirstNotEmpty(tag.fieldListMap()["GROUPING"]));
     }
 
+    if (tag.fieldListMap().contains("TRACKNUMBER")) {
+        std::pair<QString, QString> splitted(
+                TrackNumbers::splitString(toQStringFirstNotEmpty(tag.fieldListMap()["TRACKNUMBER"])));
+        if (splitted.second.isEmpty()) {
```

Comment:
I would remove this condition, and just override "second" if TRACKTOTAL or TOTALTRACKS is set. 
Since a "1/12" format is not mentioned in the Xiph specs.